project/Dependencies.scala

```diff
@@ -248,6 +248,10 @@ object Dependencies {
       kamonCore,
       kamonModule("influxdb"),
       kamonModule("system-metrics"),
+      kamonModule("status-page"),
+      kamonModule("jaeger"),
+      kamonModule("akka-http"),
```

Comment:
Do we need this module?

Reply:
yep, it contains akka + akka actor instrumentation. context propagation won't work without this one.